examples/homepage/install/arm64/pulumi_arm64.py

```diff
@@ -22,11 +22,11 @@
         pulumi.Output.concat(
             "arn:aws:lambda:",
             aws.get_region_output().name,
-            ":017000801446:layer:AWSLambdaPowertoolsPythonV2-Arm64:69",
+            ":017000801446:layer:AWSLambdaPowertoolsPythonV3-Arm64-312:69",
```

Comment:
```suggestion
            ":017000801446:layer:AWSLambdaPowertoolsPythonV3-Arm64-312:1",
```

question: version 1 like others?

Reply:
I had to update this with the new name format we decided on. It's up to date.